docs/homestakeros/2.2-configure_deploy.md

```diff
@@ -0,0 +1,98 @@
+# 2.2 Configure and Deploy
+
+This guide covers configuring your node using the HomestakerOS Web UI and deploying it to your target machine.
+
+> **Prerequisite:** Make sure you've completed the [system preparation](2.1-prepare_system.md) steps before proceeding.
+
+## Accessing the Web UI
+
+1. Start the backend and go to [https://homestakeros.com/](https://homestakeros.com/)
+2. Set the flake URI (or use the default `ponkila/homestaking-infra`)
+3. Click 'Submit' and go to the 'NixOS config' tab
+4. Create a 'new node template' from the dropdown menu
+
+## Minimal Required Configuration
+
+> **Warning:** ⚠️ Without the required settings below, you won't be able to build or access your node after deployment.
+
+At a minimum, you should configure these settings:
+
+1. **Localization**:
+   - **Hostname**: Set a name for your machine (**required**)
+   - **Timezone**: Set a valid timezone identifier from the [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+
+2. **SSH**:
+   - **Authorized Keys**: Add your public SSH key(s) to enable remote access (**required**)
+   - **Private Key File**: Set to `/mnt/nvme/secrets/ssh/id_ed25519` - this tells the system where to generate the host key
+
+## Configure Storage Mounts
+
+Configure the filesystem you created earlier:
+
+1. In the **Mounts** section, add an entry for the main mount:
+   ```
+   what    -> /dev/disk/by-label/homestaker
+   where   -> /mnt/nvme
+   type    -> ext4
+   ```
+
+## Configure Execution Client
+
+> **Note:** You must choose one execution client to pair with your consensus client.
+
+Choose and configure one execution client (e.g., Erigon):
+
+1. **Enable**: Set to `true`
+2. **Data Directory**: Set to `/mnt/nvme/erigon`
+3. **JWT Secret File**: Set to `/mnt/nvme/secrets/jwt.hex`
+4. Configure any additional options as needed
+
+> **Important:** You will need to port forward the TCP/UDP listening port on your router, which defaults to 30303 for all execution clients.
+
+## Configure Consensus Client
+
+Choose and configure one consensus client (e.g., Lighthouse):
+
+1. **Enable**: Set to `true`
+2. **Data Directory**: Set to `/mnt/nvme/lighthouse`
+3. **JWT Secret File**: Set to `/mnt/nvme/secrets/jwt.hex`
+4. **Execution Endpoint**: Ensure this matches your execution client's endpoint
+5. Configure any additional options as needed
+
+> **Important:** You will need to port forward the TCP/UDP listening port on your router, which defaults to 9000 for all consensus clients.
```

Comment:
This is not necessary if your router supports NAT-PMP or UPnP. In general, I think the firewall rules are not mandatory, but something that enhances peering (public nodes can reach you, but other "misconfigured" nodes cannot).